mw-webapp/src/globalStore/ThemeStore.ts

```diff
@@ -62,6 +62,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#A8A1DD",
     [Theme.LIGHT]: "#A8A1DD",
   },
+  hoverIconColor: {
+    [Theme.DARK]: "#A8A1DD",
+    [Theme.LIGHT]: "#8B85A1",
+  },
```

Comment:
Do we have this token in the design system

Reply:
From today, yes